client/src/Components/ReviewsContainer/Breakdowns/ProductBkdn.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react"
+import Container from "react-bootstrap/Container"
```

Comment:
I don't think it really makes a difference but if you want you could also do import { Container, Row, Col } from "react-bootstrap" so some of the imports can fit in one line.

Reply:
Thanks! I refactored to use that instead.